src/components/side-nav.tsx

```diff
@@ -10,6 +10,7 @@ import dynamic from 'next/dynamic';
 const SearchBox = dynamic(() => import('./search'));
 import { SIDEBAR_WIDTH } from '@common/constants';
 import { HEADER_HEIGHT } from '@components/header';
+import StatusCheck from './status-check';
```

Comment:
We typically prefer named exports and absolute paths:

`import { StatusCheck } from '@components/status-check';`

Reply:
Thomas beat me to it 
![image](https://user-images.githubusercontent.com/1618764/88202144-bd11e680-cc48-11ea-966e-0531258582c1.png)